StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory.swift

```diff
@@ -730,6 +732,32 @@ extension PaymentSheetFormFactory {
         return StaticElement(view: label)
     }
 
+    func makeInstantDebits() -> PaymentMethodElement {
+        return InstantDebitsPaymentMethodElement(
+            configuration: configuration,
+            titleElement: {
+                switch configuration {
+                case .customerSheet:
+                    return nil // customer sheet is not supported
+                case .paymentSheet:
+                    return makeSectionTitleLabelWith(
+                        text: STPLocalizedString(
+                            "Pay with your bank account in just a few steps.",
+                            "Instant Debits copy title for Mobile payment element form"
+                        )
+                    )
+                }
+            }(),
+            emailElement: (
+                // TODO: does this email logic make sense?
+                (configuration.billingDetailsCollectionConfiguration.email != .never)
+                ? makeEmail()
+                : nil
+            ),
```

Comment:
this might need updates

Reply:
Do you not need an email address?

Reply:
@davidme-stripe 

I wasn't sure what the logic is (`billingDetailsCollectionConfiguration`). 

I'll just make this logic be `makeEmail()` though correct me if that's wrong.

Reply:
Depends — if you always need an email address, I guess we technically would want to hide Instant Debits entirely if `billingDetailsCollectionConfiguration.email == .never`. But I'm okay with it as `makeEmail()` for now